MinPlatformPkg/Test/Library/TestPointCheckLib/DxeCheckMemoryMap.c

```diff
@@ -131,22 +131,22 @@ TestPointCheckUefiMemoryMapEntry (
     }
     Entry = NEXT_MEMORY_DESCRIPTOR (Entry, DescriptorSize);
   }
-  if (EntryCount[EfiRuntimeServicesCode] > 1) {
+  if (EntryCount[EfiRuntimeServicesCode] > 2) {
     DEBUG ((DEBUG_ERROR, "EfiRuntimeServicesCode entry - %d\n", EntryCount[EfiRuntimeServicesCode]));
   }
-  if (EntryCount[EfiRuntimeServicesData] > 1) {
+  if (EntryCount[EfiRuntimeServicesData] > 2) {

```

Comment:
Can you please define `2` in a macro? It would make the code more clear.

You can even just add it in this file if you like.

Example:

```c
#define MAX_RT_MEM_BUCKET_COUNT 2
```